<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myUI</class>
 <widget class="QWidget" name="myUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pb_icon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/tx/1.png</normaloff>:/tx/1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_lable">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>这是一个神奇的事情...</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_name">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>林间九尾</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>301</width>
     <height>471</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_friend">
    <attribute name="title">
     <string>好友列表</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-1</y>
       <width>301</width>
       <height>441</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>299</width>
        <height>439</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="wdg_friend" native="true"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>402</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_ourchat">
    <attribute name="title">
     <string>群聊</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-1</y>
       <width>301</width>
       <height>441</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>299</width>
        <height>439</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="wdg_ourchat" native="true"/>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>402</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>134</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="pb_menu">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>tabWidget</zorder>
  <zorder>pb_icon</zorder>
  <zorder>le_lable</zorder>
  <zorder>lb_name</zorder>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
